Specs/Core/CesiumTerrainProviderSpec.js

```diff
@@ -354,6 +354,29 @@ defineSuite([
     });
 
     describe('requestTileGeometry', function() {
+
+        it('uses multiple urls specified in layer.json', function() {
+            returnTileJson('Data/CesiumTerrainTileJson/MultipleUrls.tile.json');
+
+            var provider = new CesiumTerrainProvider({
+                url : 'made/up/url'
+            });
+
+            return pollToPromise(function() {
+                return provider.ready;
+            }).then(function() {
+                spyOn(loadWithXhr, 'load');
+                provider.requestTileGeometry(0, 0, 0);
+                expect(loadWithXhr.load.calls.mostRecent().args[0]).toContain('foo0.ch');
```

Comment:
This test is currently failing because the json specifies `.com` and the test expects `.ch`.  Otherwise this looks good and I'll merge it as soon as the test is fixed.